### PR TITLE
Change name of LPstyle.css to index.css. Buttons come off page when h…

### PR DESCRIPTION
**Pull request by J-coder13**
…overed over. Minor style optimizations.

Optimize  and rename index.css.

Make buttons come off page when hovered
image

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -27,6 +27,9 @@ <h2 class="center" style="font-size:40; ">Capstone Project</h2>
       </a>
 
     </div>
-    <div class="center foot" style=""><p> About.Contact Us</p></div>
+    <div class="center foot" style="">
+        <a href="https://www.merriam-webster.com/dictionary/something">About.</a>
+        <a href="https://www.google.com/">Contact us</a>
```
Comment (kev1n80) [inline src/main/webapp/index.html:32]: Thank you for doing this! I just want to make sure that anything that goes to master can be geared  towards being a final product.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -27,6 +27,9 @@ <h2 class="center" style="font-size:40; ">Capstone Project</h2>
       </a>
 
     </div>
-    <div class="center foot" style=""><p> About.Contact Us</p></div>
+    <div class="center foot" style="">
```
Comment (kev1n80) [inline src/main/webapp/index.html:30]: Is the style attribute being used here? If it's not than can you remove it, please?

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <title>Capstone Project</title>
 
     <link href="master.css" rel="stylesheet">
-    <link href="LPstyle.css" rel="stylesheet" type:"text/css">
+    <link href="index.css" rel="stylesheet" type:"text/css">
```
Comment (kev1n80) [inline src/main/webapp/index.html:6]: I love the name change!

File: src/main/webapp/index.html (hunk 2 above)
Comment (J-coder13) [reply src/main/webapp/index.html:30]: Forgot to remove that thank you!

**Pull request merged**